core/src/main/java/bio/terra/pearl/core/model/survey/SurveyResponseWithJustification.java

```diff
@@ -0,0 +1,15 @@
+package bio.terra.pearl.core.model.survey;
+
+import bio.terra.pearl.core.model.participant.Profile;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
```

Comment:
this could use a comment about how this is split from SurveyResponse since justifications aren't needed for participant flows